sync/class.jetpack-sync-module-full-sync.php

```diff
@@ -267,6 +279,33 @@ public function get_action_name( $queue_item ) {
 		return false;
 	}
 
+	public function get_action_totals( $queue_item ) {
+		if ( is_array( $queue_item ) && isset( $queue_item[1] ) && isset( $queue_item[1][0] ) ) {
```

Comment:
This is more a general comment, but it's really hard to parse what `[1][0]` is supposed to be. Maybe detail it a bit in the docblock or reference where this is documented?

Reply:
I understand the confusion - added some information in the docblock that came with  8641517ec3ea434912663488682aef82f79310a8